local-cli/core/default.config.js

```diff
@@ -22,6 +22,9 @@ const windows = require('./windows');
 const wrapCommands = require('./wrapCommands');
 const findPlugins = require('./findPlugins');
 
+const findSymlinksPaths = require('../util/findSymlinksPaths');
```

Comment:
it does work @Kureev . i moved the file to util as part of this PR given it was being used outside of "server" code.

![image](https://cloud.githubusercontent.com/assets/1494933/22441881/2df0a812-e707-11e6-9b54-38ddfaed9e10.png)